src/main/java/com/kodilla/ecommercee/GroupController.java

```diff
@@ -1,4 +1,34 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.dto.GroupDTO;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collections;
+import java.util.List;
+
+@RequestMapping("v1/groups")
+@RestController
+@CrossOrigin("*")
 public class GroupController {
```

Comment:
Czemu tu jest GroupController? Zadanie dotyczy UserController'a

Reply:
GroupController to zadanie zrobione przez Dominika. Przed pushem commita zrobiłem rebase do mastera zawierający już to zadanie zrobione